example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -171,6 +171,7 @@
 				TargetAttributes = {
 					97C146ED1CF9000F007C117D = {
 						CreatedOnToolsVersion = 7.3.1;
+						DevelopmentTeam = J22W2TNGPK;
```

Comment:
Not sure if these entires are necessary or affecting something for other contributors?

Reply:
It was removed from new commits